packages/next/src/server/app-render/use-flight-response.tsx

```diff
@@ -21,9 +21,10 @@ export function useFlightResponse(
     return flightResponseRef.current
   }
   // react-server-dom-webpack/client.edge must not be hoisted for require cache clearing to work correctly
-  const {
-    createFromReadableStream,
-  } = require(`react-server-dom-webpack/client.edge`)
+  const { createFromReadableStream } = process.env.NEXT_MINIMAL
+    ? // @ts-ignore
+      __non_webpack_require__(`react-server-dom-webpack/client.edge`)
```

Comment:
This is probably a better solution for that. Could we add this to the externals list? @feedthejim

Reply:
Yup just need to be added to webpack.config.js

Reply:
it's already in the list actually, so I'm not sure why we couldn't just clear the entry

Reply:
@sokra the problem is that webpack will create an internal module containing just the external require, causing it to be cached by that module system

Reply:
@sokra yes, but that bundle is external and never changes

Reply:
Seems this breaks app edge functions https://github.com/vercel/next.js/pull/55345

Reply:
> never changes

We clear it from the require.cache to get rid of the module scoped chunk cache, so react re-fetches chunks